vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/ConditionalFormatterTest.java

```diff
@@ -1,11 +1,20 @@
 package com.vaadin.flow.component.spreadsheet.tests;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.core.JsonParser;
+import com.nimbusds.jose.shaded.json.parser.JSONParser;
```

Comment:
These (and  a couple more) are unused imports

Reply:
Removed.